app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split(" ")
```

Comment:
You can simplify the unpacking in the way Valentyn wrote to you earlier.
command, source, destination = command.split()